### DISCUSSION TITLE
Upgrade process for baseline Digital Ocean Droplet

### DISCUSSION THREAD

**Discussion by jhkoning** [requarks/wiki#4748] [Help / Questions]
hey there, 
newbie question sorry, hopefully you'll forgive me.
my wiki.js dashboard tells me there's an upgrade, i love upgrading/updating stuff however the last time i ran a wiki.js upgrade it killed my droplet.
it is a baseline droplet, 1 cpu, 1 gb.
i'm ok with that as it'll get virtually no traffic (its for client documentation).
my question is, how can i run through the upgrade process without it killing my server?
any advice (apart from upgrading my droplet)?
many thanks,
Json

**Comment by Phil-hostwiki** [comment]
Do you mean your Digital Ocean droplet was shut down during Wiki.js upgrade? Or the Wiki.js program was shut down?
From my experience, Wiki.js upgrade isn't compute intensive.
